docs/src/inscriptions/recursion.md

```diff
@@ -3202,6 +3202,37 @@ curl -s -H "Accept: application/json" \
 ```
 </details>
 
+
+<details>
+  <summary>
+    <code>GET</code>
+    <code><b>/r/output/&lt;OUTPUT_ID&gt;</b></code>
+  </summary>
+
+### Description
+Information about an output.
+
+### Example
+```bash
+curl -s -H "Accept: application/json" \
+  http://0.0.0.0:80/r/output/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b:0
+```
+```json
+{
+  "address": "04678afdb0fe5548271967f1a67130b7105cd6a828e03909a67962e0ea1f61deb649f6bc3f4cef38c4f35504e51ec112de5c384df7ba0b8d578a4c702b6bf11d5f",
```

Comment:
This is not an address.

Reply:
yea it's strange 🤔
[https://mempool.space/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b#flow=&vout=0](https://mempool.space/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b#flow=&vout=0)